### DISCUSSION TITLE
What is the best way to get Bounding box of the polygon?

### DISCUSSION THREAD

**Discussion by DMuzer** [NetTopologySuite/NetTopologySuite#646] [Q&A] [answered]
What is the best way to get Bounding box of the polygon?

**Comment by DGuidi** [comment, marked as answer]
[Geometry.Envelope](https://nettopologysuite.github.io/NetTopologySuite/api/NetTopologySuite.Geometries.Envelope.html)

**Reply by FObermaier** [reply]
Or `Geometry.EnvelopeInternal` if you don't need a full blown Geometry.

**Reply by DMuzer** [reply]
Thank you!
